<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tlinsWPilotSettings</class>
 <widget class="QWidget" name="tlinsWPilotSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Steps defaults</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="tlinsWDecimalValue" name="stepsVX" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="tlinsWDecimalValue" name="stepsVY" native="true">
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Distance(X)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>V(Y)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Distance(Y)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[steps/s]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="tlinsWDecimalValue" name="stepsY" native="true">
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="tlinsWDecimalValue" name="stepsX" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>V(X)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[steps/s]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[steps]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[steps]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Degris defaults</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QLabel" name="label_15">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[degris/s]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Distance(X)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Distance(Y)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="tlinsWCoordinateDegris" name="degrisY" native="true">
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>V(Y)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>V(X)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="tlinsWCoordinateDegris" name="degrisVY" native="true">
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="tlinsWCoordinateDegris" name="degrisX" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="tlinsWCoordinateDegris" name="degrisVX" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[degris]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_17">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[degris/s]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_18">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[degris]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Astro defaults</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>V(RA)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="tlinsWCoordinateDeclination" name="astroVDC" native="true">
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_20">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[hours/s]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_19">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>V(DC)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_21">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[degris/s]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="tlinsWCoordinateHourAngle" name="astroVRA" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_22">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Distance(RA)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="tlinsWCoordinateHourAngle" name="astroRA" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_23">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[hours]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_24">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Distance(DC)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="tlinsWCoordinateDeclination" name="astroDC" native="true">
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_25">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[degris]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Guiding defaults</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>V(X)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_29">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[steps/s]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_26">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>V(Y)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="tlinsWCoordinateHourAngle" name="guideVX" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="tlinsWCoordinateDeclination" name="guideVY" native="true">
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_27">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Distance(X)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="tlinsWCoordinateDeclination" name="guideY" native="true">
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_28">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Distance(Y)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_30">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[steps]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="tlinsWCoordinateHourAngle" name="guideX" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_31">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[steps/s]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_32">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>[steps]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tlinsWDecimalValue</class>
   <extends>QWidget</extends>
   <header location="global">tlinswdecimalvalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tlinsWCoordinateDeclination</class>
   <extends>QWidget</extends>
   <header>tlinswcoordinatedeclination.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tlinsWCoordinateHourAngle</class>
   <extends>QWidget</extends>
   <header>tlinswcoordinatehourangle.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tlinsWCoordinateDegris</class>
   <extends>QWidget</extends>
   <header>tlinswcoordinatedegris.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
